WordPress/Tests/Files/FileNameUnitTests/some-view.inc

```diff
@@ -0,0 +1 @@
+<div><?= 'text' ?></div>
```

Comment:
This file appears to be the same as SomeView.inc, and I don't see any reference to it?

Reply:
* The contents of the file *is* the same - both files are intended to trigger the `FileName` sniff while using only short open tags.
* The filename however is **not** the same. The first file `SomeView.inc` is supposed to trigger an error. This file is there to confirm that no error is triggered when the file name complies with the basic "_lowercase with dashes_" rule. It is not mentioned in the test case file as it's not supposed to trigger an error.
* As the testcase file gathers the files to be tested using a `glob()` call, it will, however, be tested.